---
title: "magpieData"
author: 
- name: Daoyu Duan
  email: dxd429@case.edu
package: magpieData
abstract: >
 This vignette describes the usage of the package 'magpieData'. The package 'magpieData'
 is designed to provide sample MeRIP-seq data for the package 'magpie' in examples and
 vignette generation.
 These .BAM files contain eight .BAM files of chromosome 15 data of the dataset (GSE46705)
 from a study investigating METTL3-METTL14 complexmediates mammalian nuclear RNA 
 N6-adenosine methylation. In this dataset, there are two sample types 
 from human HeLa cell line: one wild type (WT) sample and one treated sample. The 
 treatment corresponds to the knockdown (KD) of complex METTL3. Each sample contains two replicates. 

output: rmarkdown::html_vignette
vignette: >
  %\VignetteIndexEntry{magpieData}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---
\tableofContents

# Usage
To get the system path to the .BAM files and corresponding annotation file, use function `getBAMpath()` in magpieData.
```{r setup}
library(magpieData)
BAM_path <- getBAMpath()
BAM_path
```
The data folder path can be then used in the package magpie during examples and vignette generation.

# Data source
The dataset (GSE46705) was downloaded from Gene Expression Omnibus (GEO) DataSets. We only extracted chromosome 15 data for this package. The complete .BAM files can be obtained from https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE46705)

# Session info {.unnumbered}
```{r sessionInfo, echo=FALSE}
sessionInfo()
```
